sdkconfig.defaults

```diff
@@ -29,3 +29,7 @@ CONFIG_SPIRAM_MODE_OCT=y
 # Enable Compiler Optimization
 CONFIG_COMPILER_OPTIMIZATION_PERF=y
 CONFIG_COMPILER_OPTIMIZATION_ASSERTIONS_DISABLE=y
+
+#increase pthread stack allocations for esp_usrsctp
+CONFIG_PTHREAD_TASK_STACK_SIZE_DEFAULT=8192
+CONFIG_PTHREAD_STACK_MIN=8192
```

Comment:
No need to modify the min stack. One may still need to pthread with smaller stack via `esp_pthread` configuration.

Reply:
Nit pick: please add a new line